src/test/java/core/basesyntax/service/ParserServiceImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static ParserService parserService;
+
+    @BeforeAll
+    static void setUp() {
+        parserService = new ParserServiceImpl();
+    }
+
+    @Test
+    void parseTransaction_WithNullArgument_notOk() {
```

Comment:
```suggestion
    void parseTransaction_withNullArgument_notOk() {
```
fix the same issue in all places